### PR TITLE
Crud customer

### PR DESCRIPTION
**Pull request by jignesh-tps**

### CODE REVIEW

File: src/app/customer/customer-form/customer-form.component.html [hunk 1]
```diff
@@ -135,127 +127,48 @@ <h1 class="page-title">{{title}}</h1>
...
-                                    </label>
-                                </div>
-                            </div>
-                        </div>-->
-
-                        <!--<div class="form-group col-sm-6 col-xs-12">
-                            <div class="col-md-4 col-sm-12 col-xs-12">
                                 <label class="custom-form-label">Billing Notification</label>
                             </div>
                             <div class="col-md-8 col-sm-12 col-xs-12">
```
Comment (jignesh-tps) [inline src/app/customer/customer-form/customer-form.component.html]: Need to Decompose this ?

File: src/app/customer/customer-form/customer-form.component.html [hunk 2]
```diff
@@ -330,70 +243,25 @@ <h1 class="page-title">{{title}}</h1>
...
-                                                <input type="text" class="form-control custom-form-control" placeholder="Enter sprint cleaning price here">
-                                            </div>
-                                        </div>
-                                        <div class="form-group col-sm-6 col-xs-12">
-                                            <div class="col-md-4 col-sm-12 col-xs-12">
-                                                <label class="custom-form-label">Deep Cleaning</label>
-                                            </div>
-                                            <div class="col-md-8 col-sm-12 col-xs-12">
-                                                <input type="text" class="form-control custom-form-control" placeholder="Enter deep cleaning price here">
+                                                <input type="text" name="customers_service_prices_attributes[price" class="form-control custom-form-control" placeholder="Enter {{cspa.service_name}} services price here" [value]="cspa.price" (input)="cspa.price = $event.target.value;">
```
Comment (jignesh-tps) [inline src/app/customer/customer-form/customer-form.component.html]: I remembered your suggestion for decompose this to different component. :+1:

Comment (sagarrabadiya) [inline src/app/customer/customer-form/customer-form.component.html]: value of name property is incorrect missing closing bracket.

File: src/app/customer/customer-form/customer-form.component.ts [hunk 3]
```diff
@@ -14,51 +14,36 @@ import { CustomerService } from '../shared/customer.service';
...
+  ) { }
 
   ngOnInit() {
     var id = this.activateRoute.params.subscribe(params => {
       var id = params['id'];
       this.title = id ? 'Edit Customer' : 'Add New Customer';
 
+      this.customerService.getServices(id)
+      .subscribe(
+        (customerServices: CustomerServicePriceAttributes[]) => this.customer.customers_service_prices_attributes = customerServices,
```
Comment (sagarrabadiya) [inline src/app/customer/customer-form/customer-form.component.ts]: never do this,
subscribe inside subscribe, this kind of thing should be done using switchMap

Comment (jignesh-tps) [inline src/app/customer/customer-form/customer-form.component.ts]: I update this code please verify.

**Pull request merged**